requirements/dev.txt

```diff
@@ -1,97 +1,216 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.6
 # To update, run:
 #
-#    pip-compile --output-file requirements/dev.txt requirements/dev.in
+#    pip-compile --output-file=requirements/dev.txt requirements/dev.in
 #
--e git+git://github.com/kytos/kytos.git#egg=kytos
--e git+git://github.com/kytos/python-openflow.git#egg=python-openflow
--e git+git://github.com/kytos/of_core.git#egg=kytos-of-core
```

Comment:
@italovalcy these `git+git://.*` were the main cause being addressed, so they got replaced by `git+https://.*` 

```
Obtaining kytos from git+git://github.com/kytos/kytos.git#egg=kytos (from -r requirements/dev.txt (line 7))
  Cloning git://github.com/kytos/kytos.git to ./.tox/py36/src/kytos
  Running command git clone --filter=blob:none -q git://github.com/kytos/kytos.git /tmp/flow_manager/.tox/py36/src/kytos
  fatal: remote error:
    The unauthenticated git protocol on port 9418 is no longer supported.
  Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
```

The rest that of the changes in the requirement I had to made since `pip-compile` wasn't being able to generate because of `flask` bringing conflicting deps as mentioned in the previous comment fyi.

Reply:
Ahh okay! great! thanks!

Reply:
A similar issue we've got on `kytos` core https://github.com/kytos-ng/kytos/pull/142/commits/df6cd593ee4c3a53b7d3d120d1f2d88d0ae9d666, https://github.com/kytos-ng/kytos/pull/142